www/searchutil.php

```diff
@@ -811,28 +811,28 @@ function doSearch($db, $term, $searchType, $sortby, $limit, $browse)
 
     case "list":
         $sortList = array(
+            'new' => array('moddate desc,', 'Newest First'),

```

Comment:
In your other changes, Oldest First follows Newest First, but not in this change.

Reply:
I noticed the Competitions browse list uses "Most Recent First". Suggest changing it to "Newest First" to be consistent.